### ISSUE TITLE
NotImplementedError: Extending this list won't work

### ISSUE THREAD

**Issue by adityabandla** [antismash/antismash#215]
Manually installed antiSMASH v5.1 on a linux server
Did a test run with the minimal flag and the output looked fine

But running a full-featured run throws the following error
```
Process ForkPoolWorker-62:
Traceback (most recent call last):
File "miniconda3/envs/antismash/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap self.run()
File "miniconda3/envs/antismash/lib/python3.7/multiprocessing/process.py", line 99, in run
self._target(*self._args, **self._kwargs)
File "miniconda3/envs/antismash/lib/python3.7/multiprocessing/pool.py", line 110, in worker
task = get()
File "miniconda3/envs/antismash/lib/python3.7/multiprocessing/queues.py", line 354, in get
return _ForkingPickler.loads(res)
File "miniconda3/envs/antismash/lib/python3.7/site-packages/antismash/common/secmet/qualifiers/nrps_pks.py", line 103, in extend
raise NotImplementedError("Extending this list won't work")
NotImplementedError: Extending this list won't work
```

**Comment by SJShaw** [comment]
We've completed thousands of runs without seeing this, so I suspect it's something specific in your environment (or possibly your input). If you rerun on a single core (`--cpus 1`), that will give us a traceback that isn't interrupted by the multiprocessing portion.

Does this happen on every input or just one? If it's every input, then it's definitely something in the environment. If it's not on every input, and you are able to share the input you used, that would be useful. If you can't share it, maybe you could trim it down to something that still causes the issue but you would be able to share.

**Comment by adityabandla** [comment]
It seems to happen with all the files.
With --cpus 1, the run finishes as expected

**Comment by SJShaw** [comment]
That it functions when not using the multiprocessing means it's something to do with the multiprocessing package. Those two warnings listed are unrelated.

The web version runs on Debian Jessie with Python 3.5.3, which is the same as my usual development machine. However, we also run on a variety of other systems and configs:
Ubuntu and Python 3.6.7,
Mac with some Python 3.7 version that I can't check right now,
Debian with Python 3.7.3

None of them have this issue. None of them use conda of any variant, so possibly that's the part that is causing the issue.

What specific version of python are you running?
Does this miniconda environment you're using have anything that would override the standard library multiprocessing module?

**Comment by CatarinaCarolina** [comment]
Having the same issue on a linux server with a conda installation of AS 5.1.1 

Are there any conda features/packages you recommend checking/adjusting?

python version is 3.7.6

**Comment by SJShaw** [comment]
There's more up-to-date info in #237, but this is fixed in 5.1.2 and it's now waiting on bioconda to update.
